script.js

```diff
@@ -0,0 +1,80 @@
+const speakers = [
+  {
+    imgSrc: './assets/mobile/kilnam.png',
+    speakerName: 'Yochai Benkler',
+    speakerField: 'Berkan Professor of History with over 20 years experience in podcast',
+    speakerDesc: 'Benkler studies commons-based peer production and published his seminal book Journey to the podcasting world.',
+  },
+  {
+    imgSrc: './assets/mobile/yochai.png',
+    speakerName: 'Kilnam Chon',
+    speakerField: '',
+    speakerDesc: 'Kilnam Chon helped bring the podcast submit to Asia and is an outspoken advocate for open podcast and digital commons',
+  },
+  {
+    imgSrc: './assets/mobile/sohyeong.png',
+    speakerName: 'SohYeong Noh',
+    speakerField: 'Director of Art Center and a board member of PD Podcasting',
+    speakerDesc: 'As the main venue for new media art production in korea, Nabi promotes cross-displinary collaboration and understanding',
+  },
+  {
+    imgSrc: './assets/mobile/julia.png',
+    speakerName: 'Julia Leda',
+    speakerField: 'President of young pirates of Europe',
+    speakerDesc: 'European Interation, political democracy and participation of youth through online as her major condem',
+  },
+  {
+    imgSrc: './assets/mobile/lila.png',
+    speakerName: 'Lila Tretikov',
+    speakerField: 'Executive Director of the media foundation',
+    speakerDesc: 'Lila Tretikov is the executive director of wikimedia foundation, the non profit organisation that operates wikipedia',
+  },
+  {
+    imgSrc: './assets/mobile/reagan.png',
+    speakerName: 'Ryan Merkley',
+    speakerField: 'CEO of PD Podcast',
+    speakerDesc: 'Ryan has been leading open source project at the mozilla foundation such as the open source movement.',
+  },
+];
+
+const speakersList = document.querySelector('.speakers-list');
+for (let i = 0; speakersList && i < speakers.length; i += 1) {
+  const div = document.createElement('div');
+  div.className = 'speaker';
+
+  const img = document.createElement('img');
+  img.src = speakers[i].imgSrc;
+  img.setAttribute('alt', 'speaker'.concat(i + 1));
+
+  const article = document.createElement('article');
+  const h3 = document.createElement('h3');
+  h3.className = 'speaker-name';
+  h3.textContent = speakers[i].speakerName;
+  const h4 = document.createElement('h4');
+  h4.className = 'speaker-field';
+  h4.textContent = speakers[i].speakerField;
+  const pDot = document.createElement('p');
+  pDot.textContent = '.....';
+  const p = document.createElement('p');
+  p.className = 'speaker-desc';
+  p.textContent = speakers[i].speakerDesc;
+
+  article.append(h3);
+  article.append(h4);
+  article.append(pDot);
+  article.append(p);
+
+  div.append(img);
+  div.append(article);
+  speakersList.append(div);
```

Comment:
- [OPTIONAL] When  to use `append()` and `appendChild()` methods. append() and appendChild() are two popular methods used to add elements into the Document Object Model(DOM). They are often used interchangeably without much troubles. However, Element.append() can append several nodes and strings, whereas Node.appendChild() can only append one node.
In your own case, you can call the append() method on the article element and then list all the nodes you want to append inside the method like this: 
`article.append(h3, h4, pDot, p)` respectively.
You can find more info about it [here](https://dev.to/ibn_abubakre/append-vs-appendchild-a4m) and [this](https://developer.mozilla.org/en-US/docs/Web/API/Element/append) too.

Reply:
Thank you very much for this feedback, wasn't aware before. I will adjust it.

Reply:
Good :+1: